arch/arm/core/aarch32/cortex_a_r/Kconfig

```diff
@@ -87,6 +87,13 @@ config CPU_CORTEX_R7
 	help
 	  This option signifies the use of a Cortex-R7 CPU
 
+config CPU_CORTEX_R52
+	bool
+	select CPU_CORTEX_R
+	select AARCH32_ARMV8_R
```

Comment:
So I will turn this into CPU_AARCH32_CORTEX_R once #43040 is merged
@bbolen @microbuilder